test/DacpacTool.Tests/PackageBuilderTests.cs

```diff
@@ -47,12 +46,27 @@ public void AddInputFile_FileExists()
             packageBuilder.UsingVersion(SqlServerVersion.Sql150);
 
             // Act
-            packageBuilder.AddInputFile(new FileInfo("../../../../TestProject/Tables/MyTable.sql"));
+            var result = packageBuilder.AddInputFile(new FileInfo("../../../../TestProject/Tables/MyTable.sql"));
 
             // Assert
+            result.ShouldBe(true);
             packageBuilder.Model.GetObject(Table.TypeClass, new ObjectIdentifier("dbo", "MyTable"), DacQueryScopes.Default).ShouldNotBeNull();
         }
 
+        [TestMethod]
+        public void AddInputFile_FileExists_WithInvalidSyntax()
+        {
+            // Arrange
+            var packageBuilder = new PackageBuilder();
+            packageBuilder.UsingVersion(SqlServerVersion.Sql150);
+
+            // Act
+            var result = packageBuilder.AddInputFile(new FileInfo("../../../../TestProjectWithExceptions/Tables/MyTable.sql"));
+
+            // Assert
+            result.ShouldBe(false);
```

Comment:
Is it possible to test the exceptions that are thrown, as well as the result of `AddInputFile`?

Reply:
Do you mean assert what is written to the console?

Reply:
@jeffrosenberg Added assertion of console output

Reply:
;) This is kind of why we have the `IConsole` abstraction. Not entirely sure why the `PackageBuilder` class didn't get this treatment. My guess is that this is the first thing I wrote so hadn't thought about it at the time. Might be worth changing now though.